src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -667,10 +674,65 @@ trait BeamHelper extends LazyLogging {
     * @param outputDirectory output folder where full configs will be generated
     */
   private def writeFullConfigs(config: TypesafeConfig, outputDirectory: String): Unit = {
-    val configConciseWithoutJson = config.root().render(ConfigRenderOptions.concise().setFormatted(true).setJson(false))
+
+    val configResolveOptions = ConfigResolveOptions.defaults().setAllowUnresolved(true)
+
+    val templateConf = ConfigFactory
+      .parseResources("beam-template.conf")
+      .withoutPath("matsim.modules.vehicles.vehiclesFile")
```

Comment:
Why are these blocked out?

Reply:
if we are getting files from user config then we are writing files to beam output json otherwise we are ignoring them